### PR TITLE
WIP Remove unnecessary fields and names from schema

### PR DESCRIPTION
**Pull request by Bakkhos**
This is just to reverse engineer netlify cms

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -13,20 +13,10 @@ collections:
     delete: false
     files:
       - file: "src/data/imprint/imprint.json"
-        name: "imprint"
+        name: "theImprint"
```
Comment (Levino) [inline static/admin/config.yml]: Why this change?

File: static/admin/config.yml [hunk 2]
```diff
@@ -13,20 +13,10 @@ collections:
...
-          - name: "type"
-            widget: "hidden"
-            default: "imprint"
-
-          - name: "title"
-            label: "Short name"
-            description: "A name to identify this object within this collection. Should be unique within the collection."
-            widget: "hidden"
-            default: "Impressum"
-
```
Comment (Levino) [inline static/admin/config.yml]: Why can we drop this?

Comment (Bakkhos) [reply static/admin/config.yml]: The name of the file is the name of the collection entry, I think

Comment (Bakkhos) [reply static/admin/config.yml]: not entirely certain though

File: static/admin/config.yml (hunk 1 above)
Comment (Bakkhos) [reply static/admin/config.yml]: The collection is named imprint 

https://github.com/hardforkio/homepage-new/blob/244d6d265575bd002c51a8a121bf05cfc1d4f4fd/static/admin/config.yml#L10-L11

I think this the name (ID) of the first (and only) entry of the collection
https://github.com/hardforkio/homepage-new/blob/244d6d265575bd002c51a8a121bf05cfc1d4f4fd/static/admin/config.yml#L14-L16
